tests/python/relay/test_ir_parser.py

```diff
@@ -113,7 +98,7 @@ def test_comments():
         UNIT
     )
 
-@if_parser_enabled
+
 def test_int_literal():
     assert isinstance(relay.fromtext(SEMVER+"1"), relay.Constant)
     assert isinstance(relay.fromtext(SEMVER+"1").data, tvm.ndarray.NDArray)
```

Comment:
I know this isn't the point of this PR, but can we have `relay.fromtext(SEMVER + _)` as a helper function for this test file?  It's a bit of an eye sore.

Reply:
Done